### DISCUSSION TITLE
Procurement documents change date in ChangeNotice

### DISCUSSION THREAD

**Discussion by DeanFirkelj** [OP-TED/eForms-SDK#578] [Q&A] [answered]
Hi,
In case of ChangeNotice, when the change also relates to procurement documents' change, the documentation says:
The procurement documents-related elements indicate whether these documents have changed and when. For the indicator, either of the two values "true" and "false" is used; **when the value is set to "true"**, **the date the procurement documents have been changed** must be specified, else the specification of this date is not allowed.

My question relates to the value of "Procurement documents change date".  The documentation implies that the changes to the procurement documents have already happened in the past, prior to the submission & publication of change notice. In our e-Procurement system, the procurement document changes are published bundled with the change notice (which I would say is logical) - the changes are effectively visible & available to economic operators simultaneosly when the notice has been published (otherwise, what's the point of change notice?). 
What should we post as value in Procurement documents change date:
A) System date -> date of creation of new procurement documents (value before or equal to the issue date of change notice)
B) Projected value of effective publishing date of the new version of procurement documentation in the future along with the change notice - estimation of the date when the change of the documentation will be available to EOs (or public). Disclaimer - this option is not "deterministic" since the timeframe policy for publishing of the notices is not deterministic.

Regards,
Dean F.

**Comment by ferrakaop** [comment, marked as answer]
This is a business decision. If your eProcurement system waits until the Change notice is published on TED, then you could play with the preferred publication date, i.e. choose a date 2 or more working days ahead (when an OJS issue is published - see the release calendar on TED) and align your documents publication date with that.  Alternatively, change your system to publish your documents straight away, then you have the date that will be shared on the TED Change notice a couple of days later, i.e. the document change date is already known when you dispatch the notice.

**Comment by DeanFirkelj** [comment]
Thnx. It will surely be 2 days ahead due to 48-hours national publication policy in Directive, not according to the TED publication policy and work-day calendar. 
However, I must say that, for the other option you suggest, it seems that Change notice becomes really a pure formality. What is the point of post-festum & pro-forma change notice if all the relevant changes have been already published in procurement documents and available to economic operators two days before the change notice on the national portal? Surely,  at least in our case, procurement documents contain all of the data presented in the contract notice in one way or another.

**Comment by ferrakaop** [comment]
There is no obligation to publish nationally after 48 hours - this is an option provided for in the Directive.

How do you announce document changes today? Our current TED-XML notices take 5 calendar days to publish. We can speed this up with our eForms systems but we haven't managed same-day publication in the OJS yet...!

I understand your point but it's not a technical issue.
